### PR TITLE
Fix and Cleanup test.py

### PR DESCRIPTION
**Pull request by AcylSilane**

### CODE REVIEW

File: README.md
Comment (timurbazhirov) [inline README.md]: We should defer to `requirements.txt` and setup.py https://github.com/Exabyte-io/TensorMol/pull/2/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7R44 instead

File: README.md
Comment (timurbazhirov) [inline README.md]: We should add a directive for creating a virtual environment instead of using sudo, and isolate the python2 and 3 cases entirely

File: samples/test.py
Comment (timurbazhirov) [inline samples/test.py]: The file is not PEP8 formatted ( there should be spaces after array members) - use re-indent in PyCharm pls.

File: samples/test.py
Comment (timurbazhirov) [inline samples/test.py]: It might be better to isolate these to a dictionary at the top and just do one `tm.PARAMS` assignment here

File: setup.py
Comment (timurbazhirov) [inline setup.py]: We should update the url, email too

File: README.md
Comment (AcylSilane) [reply README.md]: requirements.txt is gone, readme is updated to full-send  the `pip install -e .` route

File: README.md
Comment (AcylSilane) [reply README.md]: Updated readme to avoid the use of sudo.

File: samples/test.py
Comment (AcylSilane) [reply samples/test.py]: Autoformatted in PyCharm

File: setup.py
Comment (AcylSilane) [reply setup.py]: Ah! I've finally found a use for the `maintainer` field that distutils allows for. Updated with that info.

File: samples/test.py
Comment (AcylSilane) [reply samples/test.py]: Moved to the top in the most recent commit. Luckily tf.PARAMS inherits from dicts, so I could take advantage of the update() method.

**Review by timurbazhirov — APPROVED** [review]

**Pull request merged**
